learn/react-native/app-loader.md

```diff
@@ -112,25 +143,79 @@ className: headerLabel
 }
 ```
 
+#### Skeleton Loader in Container Type Widgets
+
+Container type widgets are recently getting supported by Skeleton Loader. This category includes the below widgets,
+
+- Grid Layout
+- Flex Layout
+- Accordion
+- Tabs
+- Wizard
+- Panel
+- Container
+- Tile
+
+The widgets in this category have child widgets on which the Skeleton Loader is not applied by default. To apply it on the child widgets within the Container type widget, set **`showskeletonchildren`** property as `true` in Markup script.
+
+#### Customization using Lottie Animation
```

Comment:
Need code snippet and instructions to apply clearly. Use images and video in this case.